File: Listas de Exercícios/C/Lista 1 - Casa/7.c
#include <stdio.h>
#include <math.h>

int main ()
{
    float r, r_2, r_3, V, A;
    
    printf("Volume e área da superfície da esfera\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");

    printf("Raio da esfera: "); scanf("%f", &r);
    
    r_2=pow(r, 2);
    r_3=pow(r, 3);
    
    V=(4*3.14*r_3)/3;
    A=4*3.14*r_2;
    
    printf("\nVolume: %.2f(medida)³\nÁrea (Superfície): %.2f(medida)²\n", V, A);
}

File: Listas de Exercícios/Python/Python/Lista 1 - Casa/7.py
print("Vejamos o volume e a área da superfície de uma esfera\n")

print("--Valores decimais devem ser digitados com (.)!--")

r=float(input("Raio da esfera: "))

V=4/3*3.14*r**3; V="{:.2f}".format(V)
A=4*3.14*r**2; A="{:.2f}".format(A)

print(f"\nVolume: {V}(medida)³\nÁrea (Superfície): {A}(medida)²\n")